litmus-portal/cluster-agents/subscriber/pkg/objects/objects.go

```diff
@@ -0,0 +1,297 @@
+				statefulSetList, err := GetStatefulSetsByNamespace(namespace.GetName(), clientset)
+				if err != nil {
+					panic(err.Error())
+				}
+				KubeObj := &types.KubeObject{
+					Namespace:       namespace.GetName(),
+					StatefulSetInfo: statefulSetList,
+				}
+				ObjData = append(ObjData, KubeObj)
+			}
+		}
+		if requestType == "daemonsets" {
+			for _, namespace := range namespace.Items {
+				daemonSetList, err := GetDaemonSetsByNamespace(namespace.GetName(), clientset)
+				if err != nil {
+					panic(err.Error())
+				}
+				KubeObj := &types.KubeObject{
+					Namespace:     namespace.GetName(),
+					DaemonsetInfo: daemonSetList,
+				}
+				ObjData = append(ObjData, KubeObj)
+			}
+		}
+		if requestType == "deploymentconfigs" {
+			for _, namespace := range namespace.Items {
+				deploymentConfigList, err := GetDeploymentConfigsByNamespace(namespace.GetName())
+				if err != nil {
+					return nil, err
+				}
+				KubeObj := &types.KubeObject{
+					Namespace:            namespace.GetName(),
+					DeploymentConfigInfo: deploymentConfigList,
+				}
+				ObjData = append(ObjData, KubeObj)
+			}
+		}
+		if requestType == "deploymentconfigs" {
+			for _, namespace := range namespace.Items {
+				deploymentConfigList, err := GetDeploymentConfigsByNamespace(namespace.GetName())
+				if err != nil {
+					return nil, err
+				}
+				KubeObj := &types.KubeObject{
+					Namespace:            namespace.GetName(),
+					DeploymentConfigInfo: deploymentConfigList,
+				}
+				ObjData = append(ObjData, KubeObj)
+			}
+		}
+		if requestType == "rollouts" {
+			for _, namespace := range namespace.Items {
+				rolloutList, err := GetRollOutsByNamespace(namespace.GetName())
+				if err != nil {
+					return nil, err
+				}
+				KubeObj := &types.KubeObject{
+					Namespace: namespace.GetName(),
+					Rollouts:  rolloutList,
+				}
+				ObjData = append(ObjData, KubeObj)
+			}
+		}
+		kubeData, _ := json.Marshal(ObjData)
+		var kubeObjects []*types.KubeObject
+		err := json.Unmarshal(kubeData, &kubeObjects)
+		if err != nil {
+			return nil, err
+		}
+		return kubeObjects, nil
+	} else {
+		return nil, errors.New("No namespace found")
+	}
+}
+
+//GetPodsByNamespace is used to get the Pod details available in the namespace.
+func GetPodsByNamespace(namespace string, clientset *kubernetes.Clientset) ([]types.PodInfo, error) {
+	podList := []types.PodInfo{}
+	pods, err := clientset.CoreV1().Pods(namespace).List(metav1.ListOptions{})
+	if err != nil {
+
+		return nil, err
+	}
+
+	for _, pod := range pods.Items {
+		podInfo := types.PodInfo{Name: pod.Name,
+			UID:               pod.UID,
+			Namespace:         pod.Namespace,
+			APIVersion:        pod.APIVersion,
+			ClusterName:       pod.ClusterName,
+			CreationTimestamp: pod.CreationTimestamp,
+			Containers:        pod.Spec.Containers,
+			NodeSelectors:     pod.Spec.NodeSelector,
+			Labels:            pod.GetLabels(),
+			//Annotations:     pod.GetAnnotations(),
```

Comment:
These comments....can these be removed or will it be required later?

Reply:
@arkajyotiMukherjee Yes, these are required. Was facing some issue while sending the data from subscriber with annotations. Will be updating it later.